### ISSUE TITLE
Combobox: aria-activedescendant not set

### ISSUE THREAD

**Issue by wkeese** [ibm-js/deliteful#610]
aria-activedescendant should be set to the id of the currently selected dropdown choice, but instead it's an empty string.

Can be seen in deliteful/samples/Combobox.html, where the drop down items don't have id's at all.

Note that in delteful/tests/functional/Combobox-decl.html, the dropdown items do have ids, because they are added manually within the test file.
